src/redux/actions/concepts/dictionaryConcepts.js

```diff
@@ -133,8 +133,13 @@ export const fetchDictionaryConcepts = (
   }
   try {
     const response = await instance.get(`${url}&includeRetired=1`);
-    dispatch(getDictionaryConcepts(response.data, FETCH_DICTIONARY_CONCEPT));
-    dispatch(paginateConcepts(response.data));
+    // this filter is to account for situations where a ref to a concept
```

Comment:
Instead of this comment, shouldn't we have a ticket?

Reply:
We created a ticket on the main OCL Issues repository([here](https://github.com/OpenConceptLab/ocl_issues/issues/115)). This comment in particular is just to give whoever is working on that part of the code base context as to why this was necessary.
Is it unnecessary in your opinion?

Reply:
Fair enough 😊

Reply:
Ahh yes. Adding that.

Reply:
Added.